### ISSUE TITLE
ANTLR Extension stopped working

### ISSUE THREAD

**Issue by busile** [mike-lischke/vscode-antlr4#145]
Hello.
I haven't use the extension for 2 months or so. Nothin changed in VSCode and repo (at least not intendedly), and the ANTLR extension stopped working.
Version: 2.2.4

what is not working:
1. when saving a G4 file, code not generated
2. When pressing the debug button, nothin happens. Nothing written to the log.
3. I mess up a G4 file on purpose, but no syntax errors

Settings example (all locations are valid):
{
    "antlr4.generation": {
        "mode": "external",
        "language": "CSharp",
        "outputDir": "../output_source",
        "package": "TP.Parsers"
    }
}

Configuration example:
"configurations": [
        {
            "name": "PSets grammar",
            "type": "antlr-debug",
            "request": "launch",
            "input": "antlr_inputs/PSets.txt",
            "grammar": "grammars/PSets.g4",
            "startRule": "psets",
            "printParseTree": true,
            "visualParseTree": true
        },

**Comment by mike-lischke** [comment]
Is the extension active at all? You can see that already from the syntax highlighting. Is it still installed?

**Comment by busile** [comment]
Hi Mike. Yes. It's activated. The syntax is highlighted.
I tried to uninstall and install, install older version of the extension or vs code, but no luck.

**Comment by mike-lischke** [comment]
Do you see anything in the .antlr folder in your workspace? This is where the extension stores generated files and other data.

**Comment by busile** [comment]
There was an old .antlr folder in the workspace. I deleted it and tried to save again a g4 file or run again a transformation on an existing text file, but nothing happened. The .antlr folder was not created again.
I remember when it worked, I saw a little animation in the info bar, but it's not running anymore.

The weird thing is that I tried this also on another computer, and I get the same behavior - stopped working.

**Comment by mike-lischke** [comment]
Weird behavior. You still have Java installed?

**Comment by busile** [comment]
>java -version
java version "15.0.2" 2021-01-19
Java(TM) SE Runtime Environment (build 15.0.2+7-27)
Java HotSpot(TM) 64-Bit Server VM (build 15.0.2+7-27, mixed mode, sharing)

are there any log files i can share maybe to help to shed some light on the issue?

**Comment by busile** [comment]
Hi @mike-lischke , anything else.i can send you to understand the issue?

**Comment by busile** [comment]
Hi @mike-lischke , sorry to bug you, I'm afraid I'm kind of stuck. Is there anything I can do to help resolving this issue?

**Comment by mike-lischke** [comment]
I don't see that problem here, so it's tricky to do anything with it. To me it looks as if the extension is not running, but you say it is.

**Comment by busile** [comment]
Ok, so i got some news. I forgot I created another folder when I just start playing with the ANTLR VS Code extension.
Apparently, the extension is working there (same version of VS Code and ANTLR extension as the non working folder).
So, at least I know it's working on the computer, and it might be something with the repo, code, whatever.
I'll try to isolate to see if I managed to root cause the culprit. 

Thanks.
Busi
